lib/parliament/api_v1.rb

```diff
@@ -389,7 +389,6 @@ class APIv1 < Sinatra::Base
     private
 
     def respond(path)
-      puts path
```

Comment:
Maybe we should add some logging that says something like 'Serving file: #{PATH}'?